libs/core/coroutines/include/hpx/coroutines/detail/coroutine_accessor.hpp

```diff
@@ -29,12 +29,17 @@
 
 #pragma once
 
+#include <hpx/config.hpp>
+
+#include <utility>
+
 namespace hpx { namespace threads { namespace coroutines { namespace detail {
 
     struct coroutine_accessor
     {
         template <typename Coroutine>
-        static typename Coroutine::impl_ptr get_impl(Coroutine& x)
+        HPX_FORCEINLINE static typename Coroutine::impl_ptr get_impl(
```

Comment:
Not your change, but why is this static?

Reply:
Because it does not need to access any members of that struct.

Reply:
D'oh. I missed that it was inside a struct... Makes absolute sense.